pkg/config/config.go

```diff
@@ -110,40 +152,69 @@ func NewLocalConfig() LocalConfig {
 	}
 }
 
+// NewProxyLocalConfig creates an empty ProxyLocalConfig struct with typeMeta populated
+func NewProxyLocalConfig() ProxyLocalConfig {
```

Comment:
can we unexport this, as it shouldn't be used outside of this package

Reply:
done